site/content/contribute/mobile/developer-setup.md

```diff
@@ -156,6 +156,25 @@ In order to develop and build the Mattermost mobile apps you'll need to get a co
 
 ## Troubleshooting
 
+### Errors When Running 'make run'
```

Comment:
this should be when running the react native packager

Reply:
Thanks, done update and moved it at the bottom.
![screen shot 2018-09-20 at 2 01 45 pm](https://user-images.githubusercontent.com/5334504/45798856-fc362000-bcdd-11e8-8017-9a923618c3eb.png)